backend/api/Areas/Project/Models/Dispose/ProjectModel.cs

```diff
@@ -265,14 +275,9 @@ public class ProjectModel : Api.Models.BaseModel
         public decimal NetBook { get; set; }
 
         /// <summary>
-        /// get/set - A static netbook value used on the close out form.
+        /// get/set - The market value.
         /// </summary>
-        public decimal? CloseOutNetbook { get; set; }
-
-        /// <summary>
-        /// get/set - The estimated value.
-        /// </summary>
-        public decimal Estimated { get; set; }
+        public decimal Market { get; set; }
```

Comment:
All `Estimate` properties have been renamed to `Market`.  This is because it better reflects the purpose and reduced confusion.  There were over three different labels which all were the same field (Estimated Market Value, Current Market Value, Sales Proceeds).